exonum/src/blockchain/schema.rs

```diff
@@ -171,9 +171,9 @@ impl<T: Access> Schema<T> {
         pool.get().unwrap_or(0)
     }
 
-    /// Returns a table that represents a set of uncommitted transactions hashes.
-    pub fn transactions_pool(&self) -> KeySetIndex<T::Base, Hash> {
-        self.access.get_key_set(TRANSACTIONS_POOL)
+    /// Returns a table that represents a set of uncommitted transactions.
```

Comment:
Nit: Maybe, rewrite as "Returns a table with uncommitted transactions, where the hash of the transaction is mapped to its contents"?